src/providers/Global.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import GlobalProvider from './Global.provider';
+import { BrowserRouter } from 'react-router-dom/cjs/react-router-dom.min';
+
+const mockValue = {
+  login: jest.fn(),
+  logout: jest.fn(),
+  toggleTheme: jest.fn(),
+  handleChange: jest.fn(),
+};
+
+//this test dont up the coverage
+describe('<GlobalProvider>', () => {
+  test('should create the GlobalProvider element correctly', () => {
+    const { container } = render(
+      <BrowserRouter>
+        <GlobalProvider value={mockValue} />
+      </BrowserRouter>
+    );
+
+    expect(container).toBeValid();
+  });
+});
```

Comment:
For testing this we could check that the components that consume the context actually display the value. e.g. That the search element displays the search value stored in context